EIPS/eip-5069.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 5069
+title: EIP Editor Handbook
+description: Formalizes the process for becoming an EIP editor
+author: Pooja Ranjan (@poojaranjan), Pandapip1 (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/pr-5069-eip-editor-handbook/9137
+status: Living
+type: Informational
+created: 2022-05-02
+requires: 1
+---
+
+## Abstract
+An Ethereum Improvement Proposal (EIP) is a design document providing information to the Ethereum community, or describing a new feature for Ethereum or its processes or environment. The EIP standardization process is a mechanism for proposing new features, for collecting community technical input on an issue, and for documenting the design decisions that have gone into Ethereum. Because improvement proposals are key components of Ethereum blockchain, it is important that they are well reviewed before reaching `Final` status. EIPs are stored in text files in a versioned repository which is monitored by the EIP editors.
+
+This EIP describes the recommended process for becoming an EIP editor.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+### Application and Onboarding Process
+Any contributor of Ethereum ecosystem, having a good understanding of EIP standardization and network upgrade process, intermediate level experience on the core and/or application side of the Ethereum blockchain, and willingness to contribute to the process management MAY apply to become an EIP editor. Potential EIP editors SHOULD have the following skills:
+- Good communication skills
+- Ability to handle contentious discourse
+- 1-5 spare hours per week
+
+The best available resource to understand the EIP process is [EIP-1](./eip-1.md). Anyone desirous of becoming an EIP editor MUST understand this document. Afterwards, participating in the EIP process by commenting on and suggesting improvements to PRs and issues will familliarize the procedure, and is RECOMMENDED. The contributions of newer editors SHALL be monitored by other EIP editors.
+
+A pull request with a request to be listed as EIP editor MAY be made by anyone who feels comfortable with the EIP process. If the existing EIP editors agree, the author SHALL become a full EIP editor. This SHALL notify the editor of relevant new proposals submitted in the EIPs repository, and they SHALL review and merge pull requests. Once the first PR is merged, another pull request to update the editor list SHALL be made.
```

Comment:
```suggestion
Anyone meeting the above requirements MAY make a pull request adding themselves as an EIP editor. If the existing EIP editors approve, the author SHALL become a full EIP editor. This SHALL notify the editor of relevant new proposals submitted in the EIPs repository, and they SHALL review and merge pull requests. Once the first PR is merged, another pull request to update the editor list SHALL be made.
```

Changed the wording a bit, but I'm still not sure what the second pull request is for?